content/en/docs/instrumentation/python/automatic.md

```diff
@@ -88,6 +92,15 @@ The examples that follow send instrumentation results to the console. Learn more
 about installing and configuring the [OpenTelemetry Distro](../distro) to send
 telemetry to other destinations, like an OpenTelemetry Collector.
 
+**Note**: If you were to use auto-instrumentation and manually instrumenting your code the
+pacakage `opentelemetry-distro` won't let you set a `Resource` with attributes defined in your code, 
+you will need to set all your desired parameters through [environment variables][env]
```

Comment:
I'm not sure if this is correct. `opentelemetry-instrument` command accept CLI arguments that allow users to configure almost everything

Reply:
Good point, you can set things through the cli command as well.

Reply:
Given that the autoinstrumentation route can handle lots of customization, I'm trying to compile a list of benefits of manual instrumentation OVER auto-instrumentation. I think that would help make the paragraph clearer if it begins with something like "Manual Instrumentation and Auto-instrumentation are mutually exclusive and are not meant to be combined. Here are some differences to help you choose between them..." So, any ideas on the benefits of manual instrumentation?

Reply:
@jeremydvoss Hmmm, that would not be correct. Manual instrumentation and automatic instrumentation are not mutually exclusive. It's Automatic instrumentation via the agent and automatic instrumentation via an _instrumentation library_ that are considered mutually exclusive. Manual instrumentation - instrumenting your own code - will always be valid and valuable no matter how you choose to use automatic instrumentation.